Aries/Aries/Admin/Proofs/AdminProofs.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  AdminProofs.swift
+//  Aries
+//
+//  Created by David Clawson on 12/14/21.
+//
+
+import Foundation
+
+
+public class AdminProofs {
+    
+    private var messageSender: MessageSender
+    private var adminConnection: ConnectionRecord
+
+    public init (messageSender: MessageSender, adminConnection: ConnectionRecord){
+        self.messageSender = messageSender
+        self.adminConnection = adminConnection
+    }
+
+    private func setAdminConnection(adminConnection: ConnectionRecord){
+        self.adminConnection = adminConnection
+    }
+    
+    //TODO - PresentationsListMessage does not contain thread information (should I fix this?)
+    public func sendGetPresentations(){
+        let message = PresentationsGetListMessage(connectionId: "") //Figure out paramater
+        self.messageSender.sendMessage(message: message, connectionRecord: adminConnection)
+    }
```

Comment:
```suggestion
    public func sendGetPresentations(connectionId: String? = nil){
        let message = PresentationsGetListMessage(connectionId: connectionId) //Figure out paramater
        self.messageSender.sendMessage(message: message, connectionRecord: adminConnection)
    }
```
connectionId parameter is optional but should default to nil/null when not specified, that way we will receive presentations for all connections.

Reply:
Fixed, I think. I had to change PresentationsGetListMessage to work with an optional. Swift requires inits to have a value for all stored properties. If there's an optional, it must be a "var" and not a "let".